mteb/tasks/PairClassification/multilingual/XNLI.py

```diff
@@ -103,19 +103,22 @@ def dataset_transform(self):
     "odiya": ["ory-Orya"],
     "sanskrit": ["san-Deva"],
     "tamil": ["tam-Taml"],
+    "turkish": ["tur-Latn"],
+    "greek": ["ell-Grek"],
+    "russian": ["rus-Cyrl"],
 }
 
 
 class XNLI2(XNLI):
     metadata = TaskMetadata(
         name="XNLI2",
         dataset={
-            "path": "mteb/xnli2.0-multi-pair",
-            "revision": "1a3794e88c74db2a13aebc5e8abc3dc4312a7543",
+            "path": "Andrianos/xnli2.0-multi-pair",
```

Comment:
Stepping in before merging, don't you think it would be better to update the current mteb dataset with these new languages?